modules/api-svc/src/lib/model/InboundTransfersModel.js

```diff
@@ -875,15 +978,18 @@ class InboundTransfersModel {
                 }
             }
         }
-        if(this.data) {
+        if (this.data) {
             //we have persisted state so update that with this error
             this.data.lastError = {
                 originalError: err.stack || util.inspect(err),
-                mojaloopError: mojaloopError,
+                mojaloopError,
             };
             this.data.currentState = SDKStateEnum.ERROR_OCCURRED;
-            await this._save();
+            this.data.transferId
```

Comment:
Do we need this condition?

Reply:
to save state either for _quote_ or for _fxQuote_,   (coz for now these methods have a slightly different impl. (in particular, different _cache keys_)